variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactory.java

```diff
@@ -59,7 +59,9 @@
 
         String chromosome = getChromosomeWithoutPrefix(fields);
         int position = getPosition(fields);
-        Set<String> ids = getIds(fields);
+        //EVA-942 - Ignore IDs submitted through VCF
+        //Set<String> ids = getIds(fields);
```

Comment:
It's ok to leave an explanation, but don't keep old code:
```
Set<String> ids = new HashSet<>();    //EVA-942 - Ignore IDs submitted through VCF
```

Reply:
👍